eve/workers/pod-builder/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos/7
+
+COPY requirements-eve.txt /tmp/
+
+RUN yum install -y \
+    python2-pip \
+    genisoimage \
+    git \
+    && curl -fsSL https://get.docker.com/ | sh
```

Comment:
careful when installing the docker client like that. I really don't recommend it, you don't control the docker version, so everyone and even the CI will have a slightly different image due to when they built it.